doc/lua_api.md

```diff
@@ -6533,6 +6533,13 @@ This allows you easy interoperability for delegating work to jobs.
     * Register a path to a Lua file to be imported when an async environment
       is initialized. You can use this to preload code which you can then call
       later using `minetest.handle_async()`.
+* `minetest.register_async_metatable(name, mt)`:
+    * Register a metatable that should be preserved when data is transferred
+      between the main thread and the async environment.
+    * `name` is a string that identifies the metatable.
+    * Please note that the metatable must be registered in the main environment
+      and the async environment.
+
 
 ### List of APIs available in an async environment
```

Comment:
this section should mention that `register_async_metatable` is available in the async env too

Reply:
I listed this in a later section (list of APIs in the async env)